docs/book/src/cronjob-tutorial/testdata/project/Makefile

```diff
@@ -69,17 +69,18 @@ test: manifests generate fmt vet setup-envtest ## Run tests.
 # The default setup assumes Kind is pre-installed and builds/loads the Manager Docker image locally.
```

Comment:
Could we have just one commit with all changes? 
We have a policy that is to ensure that we have a good history and each PR has just one purpose.